File: Day11/SumOfDigits.cpp
#include<iostream>
using namespace std;
int getSumOfDigit(int num){
    int sum=0;
    while(num !=0){
        sum += (num%10);
        num /= 10;
    }
    return sum;
}
int main(){
    int num;
    cout<<"Enter the number : ";
    cin>>num;
    int totalSum = getSumOfDigit(num);
    cout<<"Sum of digits: "<<totalSum;
    return 0;
}

File: Day11/SumOfDigits.java
package Day11;
import java.util.Scanner;
public class SumOfDigits {
    public static int getSumOfDigit(int num){
        int sum=0;
        while(num !=0){
            sum += (num%10);
            num /= 10;
        }
        return sum;
    }
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter the number: ");
        int num = sc.nextInt();
        sc.close();
        int totalSum = getSumOfDigit(num);
        System.out.println("Sum of digits :" + totalSum);
    }
}